docs/Makefile

```diff
@@ -18,3 +18,7 @@ help:
 # "make mode" option.  $(O) is meant as a shortcut for $(SPHINXOPTS).
 %: Makefile
 	@$(SPHINXBUILD) -M $@ "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
+
+test:
+	pytest 
```

Comment:
What's the expected benefit of this ?

Reply:
See https://github.com/Galileo-Galilei/kedro-pandera/pull/9/files#r1279762561, remove it for now.

Reply:
I merged it literally 1 mn before this message 😅 Not very important, we can clean it up later